charts/scalardb-graphql/templates/scalardb-graphql/configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "scalardb-graphql.fullname" . }}-config-file
+data:
+  # Create a database.properties file which is config file of Scalar DB server.
+  database.properties: |-
+    scalar.db.contact_points={{.Values.scalarDbGraphQlConfiguration.contactPoints}}
+    scalar.db.contact_port={{.Values.scalarDbGraphQlConfiguration.contactPort}}
+    scalar.db.username={{.Values.scalarDbGraphQlConfiguration.username}}
+    scalar.db.password={{.Values.scalarDbGraphQlConfiguration.password}}
```

Comment:
ConfigMap resource is used here to pass credential information (username and password). It is appropriate to use Secret resource as in scalardl.

https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardl/templates/ledger/deployment.yaml#L59-L76

scalardb also currently uses ConfigMap. At least for production use, Secret resource must be available for credential information.

https://github.com/scalar-labs/helm-charts/blob/main/charts/scalardb/templates/scalardb/configmap.yaml#L11-L12
https://github.com/scalar-labs/helm-charts/pull/12#discussion_r684569996

Reply:
Thank you for the review!
I added a Secret resource to store database credentials.
The style is unified with the Scalar DB Server chart: #98